lms/djangoapps/courseware/grades.py

```diff
@@ -267,6 +268,8 @@ def create_module(descriptor):
 
         totaled_scores[section_format] = format_scores
 
+    # Grading policy might be overriden by a POC, need to reset it
+    course.set_grading_policy(course.grading_policy)
```

Comment:
@pdpinch @ormsbee I'm fine saying that students in POCs don't get certificates. I would however consider the use case of a student enrolled in a POC and the MOOC that the POC is running in. Would a student in that case get a cert?